custom/intrahealth/sqldata.py

```diff
@@ -78,6 +85,10 @@ def calculate_total_row(self, rows):
                     total_row.append('')
         return total_row
 
+    @property
+    def filter_values(self):
+        return clean_IN_filter_value(super(BaseSqlData, self), 'archived_locations')
```

Comment:
forgot to call the baseclass property, made a fix for it, then accidentally pushed a62aa08a810e5cd90f2a40904ab06ebf659c2690 to master :fearful: So ... fixed now.